subjects/java/projects/travel-plan/README.md

```diff
@@ -0,0 +1,66 @@
+## Travel-Plan
+
+### Objectives
+
+The primary goal of this project is to construct a robust and scalable environment for the Travel Management System and to develop a comprehensive Admin Dashboard that serves as the control center for travel management operations.
+
+### Instructions
+
+This is the first part of the last project and for this part, the students will be asked to build the working environment and implement the admin part to ease the last part of the project.
+
+#### 1. Environment Configuration
+
+##### Configure a Microservices Architecture:
+-    Follow industry best practices for scalability and high availability.
+-    Implement multiple replicas of each microservice for load balancing and failover mechanisms.
+
+##### Database Installation and Configuration:
+-    Set up PostgreSQL and Neo4j databases.
+-    Ensure databases are containerized to facilitate scalability and replication.
+
+##### Continuous Integration/Continuous Delivery (CI/CD) Pipeline:
+-   Employ Jenkins for CI/CD and unit testing.
+-   Employ SonarQube to automate code quality checks
+
+##### Use of Docker and Ansible:
+-    Docker to support automated provisioning and deployment of microservices.
+-    Ansible for creating playbooks to deploy all system elements consistently and scalably.
+
+##### Outcome:
+-    Achieve an automated, scalable infrastructure.
+-    Prepare the groundwork for Admin Dashboard development and additional features of the Travel Management System.
+
+#### 2. Development and Design
+
+- Craft an Admin Dashboard enabling administrators to oversee users, travels, and payment gateways.
+- The admin should be able to add, edit or delete any user, travel, payment method and all the related parts correctly (think about database cascading update and delete).
+- Ensure each travel entry includes a destination or multiple destinations, dates, duration, activities, accommodation, and transportation details.
+- For the payment methods you can search about it to know what you need to support the most you have to support at least [Stripe](https://stripe.com/docs/development) and [PayPal](https://developer.paypal.com/home).
```

Comment:
This sentence is not very clear to me. Could the following be enough or is there something else required?

```
The Admin Dashboard should support at least [Stripe](https://stripe.com/docs/development) and [PayPal](https://developer.paypal.com/home) as payment methods.
```